### PR TITLE
Github Actions / Dobi / Docker: Change when CDK build happens at CI time. Optimize CDK dockerfile.

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
none

### What changes does this PR make to Grapl? Why?
- makes CDK its own task in CI as opposed to lumped in to integration tests and build and stuff
- optimized CDK dockerfile to cache dependencies

### How were these changes tested?
locally (✿◕‿◕)

### CODE REVIEW

File: dobi.yaml [hunk 1]
```diff
@@ -369,13 +369,16 @@ image=swarm-lifecycle-event-handler-build:
 
 # js images
 
+# Exists primarily for typechecking, though I intend to add real unit tests before 2021 ~ wimax
```
Comment (jgrillo-grapl) [inline dobi.yaml:372]: 🕙

File: src/js/grapl-cdk/.dockerignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 zips
 cdk.out
+node_modules
```
Comment (jgrillo-grapl) [inline src/js/grapl-cdk/.dockerignore:3]: 🙏

**Review by jgrillo-grapl — APPROVED** [review]
💯

File: src/js/grapl-cdk/Dockerfile [hunk 3]
```diff
@@ -3,16 +3,18 @@
 # of other services/builds.
 
 FROM node:alpine3.10 AS grapl-cdk-build
-WORKDIR /home/grapl
-COPY . .
```
Comment (wimax-grapl) [inline src/js/grapl-cdk/Dockerfile:7]: this early COPY invalidates the rest of the cache. no improvement for CI but definite improvements for local build

**Pull request merged**